code/modules/mob/living/simple_animal/hostile/netherworld.dm

```diff
@@ -47,10 +76,115 @@
 	..()
 	if(stat)
 		return
-	if(prob(10))
+	if(prob(10) && !hushed)
 		var/chosen_sound = pick(migo_sounds)
 		playsound(src, chosen_sound, 100, TRUE)
 
+/mob/living/simple_animal/hostile/netherworld/migo/Logout()
+	..()
+	hushed = FALSE
+	if(impersonation)
+		ChangeVoice()
+
+/mob/living/simple_animal/hostile/netherworld/migo/GetVoice()
+	if(impersonation)
+		return impersonation
+	return "[src]"
+
+/mob/living/simple_animal/hostile/netherworld/migo/proc/Hush()
+	set name = "Hush"
```

Comment:
As far as i can tell you don't use this as a verb anywhere so drop the set's. Same for proc below.